kong/clustering/services/sync/rpc.lua

```diff
@@ -107,48 +98,12 @@ function _M:init_cp(manager)
       return nil, err
     end
 
-    -- is the node empty? If so, just do a full sync to bring it up to date faster
     if default_namespace_version == 0 or
```

Comment:
I think we could output full sync to DP, ignoring DP's version. And if necessary, DP should not pass the version to CP, here is full sync mechanism. No version need to be considered

Reply:
keep this to make this patch simple.